modules/EnsEMBL/Web/Component/Shared.pm

```diff
@@ -1283,4 +1283,34 @@ sub render_consequence_type {
   return ($type) ? qq{<span class="hidden">$rank</span>$type} : '-';
 }
 
+sub render_evidence_status {
+  my $self      = shift;
+  my $evidences = shift;
+
+  my $render;
+  foreach my $evidence (sort {$b =~ /1000|hap/i <=> $a =~ /1000|hap/i || $a cmp $b} @$evidences){
+    my $evidence_label = $evidence;
+       $evidence_label =~ s/_/ /g;
+    $render .= sprintf('<img src="/i/val/evidence_%s.png" class="_ht" title="%s"/><span class="hidden export">%s,</span>',
```

Comment:
Hi Laurent, Could you please use static server host name ($self->img_url) for these two images? I know you just moved the code around, but it should have the static server address in the first place.